### PR TITLE
Remote access docs

### PR DESCRIPTION
**Pull request by BKristenssonAlfsson**
Made a basic sequence diagram in how the flow is going through the different files and functions for connecting.

### CODE REVIEW

**Review by ivelin — COMMENTED** [review]
@BKristenssonAlfsson would you like to update this document to reflect the latest behavior we just merged?

Also, you can link to this document from the docs menu in [this yaml](https://github.com/ambianic/ambianic-docs/blob/master/mkdocs.yml).

**Comment by BKristenssonAlfsson** [comment]
@ivelin Updated the documentation

**Review by ivelin — COMMENTED** [review]
@BKristenssonAlfsson this is great. Thank you! Do you think it belongs in the Developer or the User section of the docs menu?

File: mkdocs.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ nav:
       - AI Apps: users/ai-apps.md
       - How To: users/howto.md
       - Support: users/support.md
+      - Remote and Local pairing: developers/edge-connection.md
```
Comment (ivelin) [inline mkdocs.yml]: @BKristenssonAlfsson I see the file is now referenced in the User section, which makes sense, but it resides in the developer directory. Is that the intention?

**Comment by ivelin** [comment]
@BKristenssonAlfsson related to my earlier comment. Are you ok with renaming the file edge-pairing.md to clarify that we are talking about the initial connectivity setup and not accessing edge remotely in general.

File: mkdocs.yml (hunk 1 above)
Comment (ivelin) [inline mkdocs.yml]: Have you tested this locally? The file is now in users directory, but the link in mkdocs.yml points to a file in the dev dir.

**Pull request merged**